src/components/pages/Volunteers/ServerFailure.ts

```diff
@@ -0,0 +1,2 @@
+export const serverFailure =
```

Comment:
nit: as this is being treated as a component, I think we should capitalize "ServerFailure"